subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -31,6 +31,8 @@
   int remotePort = 23;
   int interfacePort = 12;
 
+  boolean switchSupportsPoe = true;
```

Comment:
Why/how is this always true? I don't see where the value is written anywhere. Some switches don't support PoE, so... ?

Reply:
added additional arguments